include/glow/Base/Tensor.h

```diff
@@ -25,6 +25,7 @@
 #include "glow/Support/Random.h"
 
 #include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Can you forward declare this instead? 

raw_ostream is a big header file and it can slow down the compilation because everything includes Tensor.h